integration-tests/tests/unix/unix_metrics_test.go

```diff
@@ -0,0 +1,163 @@
+		"node_memory_VmallocUsed_bytes",
+		"node_memory_WritebackTmp_bytes",
+		"node_memory_Writeback_bytes",
+		"node_netstat_Icmp6_InErrors",
+		"node_netstat_Icmp6_InMsgs",
+		"node_netstat_Icmp6_OutMsgs",
+		"node_netstat_Icmp_InErrors",
+		"node_netstat_Icmp_InMsgs",
+		"node_netstat_Icmp_OutMsgs",
+		"node_netstat_IpExt_InOctets",
+		"node_netstat_IpExt_OutOctets",
+		"node_netstat_TcpExt_ListenDrops",
+		"node_netstat_TcpExt_ListenOverflows",
+		"node_netstat_TcpExt_TCPSynRetrans",
+		"node_netstat_Tcp_InErrs",
+		"node_netstat_Tcp_InSegs",
+		"node_netstat_Tcp_OutRsts",
+		"node_netstat_Tcp_OutSegs",
+		"node_netstat_Tcp_RetransSegs",
+		"node_netstat_Udp6_InDatagrams",
+		"node_netstat_Udp6_InErrors",
+		"node_netstat_Udp6_NoPorts",
+		"node_netstat_Udp6_OutDatagrams",
+		"node_netstat_Udp6_RcvbufErrors",
+		"node_netstat_Udp6_SndbufErrors",
+		"node_netstat_UdpLite_InErrors",
+		"node_netstat_Udp_InDatagrams",
+		"node_netstat_Udp_InErrors",
+		"node_netstat_Udp_NoPorts",
+		"node_netstat_Udp_OutDatagrams",
+		"node_netstat_Udp_RcvbufErrors",
+		"node_netstat_Udp_SndbufErrors",
+		"node_network_carrier",
+		"node_network_info",
+		"node_network_mtu_bytes",
+		"node_network_receive_bytes_total",
+		"node_network_receive_compressed_total",
+		"node_network_receive_drop_total",
+		"node_network_receive_errs_total",
+		"node_network_receive_fifo_total",
+		"node_network_receive_multicast_total",
+		"node_network_receive_packets_total",
+		"node_network_speed_bytes",
+		"node_network_transmit_bytes_total",
+		"node_network_transmit_compressed_total",
+		"node_network_transmit_drop_total",
+		"node_network_transmit_errs_total",
+		"node_network_transmit_fifo_total",
+		"node_network_transmit_packets_total",
+		"node_network_transmit_queue_length",
+		"node_network_up",
+		"node_nf_conntrack_entries",
+		"node_nf_conntrack_entries_limit",
+		"node_os_info",
+		"node_sockstat_FRAG6_inuse",
+		"node_sockstat_FRAG_inuse",
+		"node_sockstat_RAW6_inuse",
+		"node_sockstat_RAW_inuse",
+		"node_sockstat_TCP6_inuse",
+		"node_sockstat_TCP_alloc",
+		"node_sockstat_TCP_inuse",
+		"node_sockstat_TCP_mem",
+		"node_sockstat_TCP_mem_bytes",
+		"node_sockstat_TCP_orphan",
+		"node_sockstat_TCP_tw",
+		"node_sockstat_UDP6_inuse",
+		"node_sockstat_UDPLITE6_inuse",
+		"node_sockstat_UDPLITE_inuse",
+		"node_sockstat_UDP_inuse",
+		"node_sockstat_UDP_mem",
+		"node_sockstat_UDP_mem_bytes",
+		"node_sockstat_sockets_used",
+		"node_softnet_dropped_total",
+		"node_softnet_processed_total",
+		"node_softnet_times_squeezed_total",
+		"node_textfile_scrape_error",
+		"node_time_zone_offset_seconds",
+		"node_timex_estimated_error_seconds",
+		"node_timex_maxerror_seconds",
+		"node_timex_offset_seconds",
+		"node_timex_sync_status",
+		"node_uname_info",
+		"node_vmstat_oom_kill",
+		"node_vmstat_pgfault",
+		"node_vmstat_pgmajfault",
+		"node_vmstat_pgpgin",
+		"node_vmstat_pgpgout",
+		"node_vmstat_pswpin",
+		"node_vmstat_pswpout",
```

Comment:
That's about 150 metrics. If they all fail that will spawn a HUGE block of text in the terminal which would make debugging hard. We should find a way to return either "all expected metrics are missing" or "the following metrics are missing:(list of the missing metrics)" if only a subset failed. You don't need to solve this in your branch but if you could add a "TODO" that would be nice